src/elements/content-sidebar/SidebarNavSignButton.scss

```diff
@@ -0,0 +1,31 @@
+@import './mixins';
+@import '../common/variables';
+
+.bcs-SidebarNavSign {
+    @include bdl-SidebarNavButton;
+}
+
+.bcs-SidebarNavSign-icon {
+    @include bdl-SidebarNavIcon;
+}
+
+.bcs-SidebarNavSign-icon--grayscale {
+    display: none;
+    margin-top: 2px; //for visual alignment
+}
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNavSign-icon {
+            display: none;
+        }
+
+        .bcs-SidebarNavSign-icon--grayscale {
+            display: inline;
+        }
+    }
+}
+
+.sign-menu-icon path {
+    fill: $bdl-gray;
+}
```

Comment:
is this needed? it looks like it's part of the `SidebarNavSign` component and not `SidebarNavSignButton`